web/src/table/MfaAccountTable.js

```diff
@@ -159,14 +178,23 @@ class MfaAccountTable extends React.Component {
         title={() => (
           <div>
             {this.props.title}&nbsp;&nbsp;&nbsp;&nbsp;
-            <Button style={{marginRight: "5px"}} type="primary" size="small" onClick={() => this.addRow(table)}>{i18next.t("general:Add")}</Button>
-            <Popover trigger="focus" content={
-              <QRCode
-                value={this.state.qrUrl}
-                icon={this.state.icon}
-                bordered={false}
-              />
-            }>
+            <Button style={{marginRight: "10px"}} type="primary" size="small" onClick={() => this.addRow(table)}>{i18next.t("general:Add")}</Button>
+            <Popover trigger="focus" overlayInnerStyle={{padding: 0}}
+              onOpenChange={(visible) => {
+                if (visible) {
+                  this.getQrUrl();
+                }
+              }}
+              content={
+                this.state.qrUrl ?
+                  <QRCode
```

Comment:
define `renderQrCode`

Inside it, first line is `const qrUrl = this.getQrUrl()`